pkg/flagset/gateway.go

```diff
@@ -153,7 +154,7 @@ func GatewayWithConfig(cfg *config.Config) []cli.Flag {
 		},
 		&cli.StringFlag{
 			Name:        "share-folder",
-			Value:       "/",
+			Value:       "Shares",
```

Comment:
This change is important, when we want to update reva.
I had to debug for a few hours because my local setup wasn't working and it all was caused by this setting.